### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by andraaspar** [smysnk/gulp-rev-all#214]
`gulp-rev-all` is the killer feature of `gulp`. The only reason I spin it up these days at all. So simple to configure, and it just works. It is the tool every web dev should know. Too few appreciate the awesomeness of this pragmatic utility. Thank you for keeping it alive.

**Comment by SamuelMiller** [comment]
I agree. The plugin is super easy to use. Hopefully, the developer will continue to update the plugin to fix some of the bugs.
